CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## Pending changes
 
-–
+– [#247](https://github.com/bumble-tech/appyx/pull/247) – **Breaking change**: Removed `DummyNavModel` from testing libraries. Use `NoOpNavModel` instead.
```

Comment:
let's not introduce any breaking api changes for 1.0

Reply:
what do you propose? wait for 1.0 to be cut? or move this model to samples for now with a plan to move it to core later?

Reply:
You do not need to remove it. Introduce type alias + deprecate notice.

Reply:
yep, will do!